tools/toit.toit

```diff
@@ -510,10 +512,21 @@ compile-or-analyze-or-run --command/string parsed/cli.Parsed:
   if command == "analyze":
     args.add "--analyze"
   else:
+    optimization/int := 0
     if parsed["optimization-level"]:
-      optimization/int := parsed["optimization-level"]
+      optimization = parsed["optimization-level"]
       if not 0 <= optimization <= 2: error "Invalid optimization level"
-      args.add "-O$optimization"
+
+    args.add "-O$optimization"
```

Comment:
So this changes things a bit. The compiler can no longer pick the default and you've changed it zero.

Reply:
So why do we have an if when we have a default value?

Reply:
Because I forgot to push my changes after the feedback...